programs/programs/co/emergency_rental_assistance/calculator.py

```diff
@@ -0,0 +1,44 @@
+from programs.programs.calc import Eligibility, ProgramCalculator
+import programs.programs.messages as messages
+from integrations.util.cache import Cache
+from programs.sheets import sheets_get_data
+from programs.co_county_zips import counties_from_zip
+
+
+class EmergencyRentalAssistanceIncomeLimitsCache(Cache):
+    expire_time = 60 * 60 * 24
+    default = {}
+
+    def update(self):
+        spreadsheet_id = '1QHb-ZT0Y2oWjFMoeP_wy8ClveslINWdehb-CXhB8WSE'
+        range_name = "'2022 80% AMI'!A2:I"
```

Comment:
Can't open the spreadsheet and verify the columns. The range `A2:I` looks a little strange. Ignore this if it's matching the spreadsheet.

Reply:
If you don't specify an end row, it will select all of the cells in the column. So this range selects all rows bellow `A2` and the columns `A` through `I`

Reply:
Gotcha, thanks!